packages/clerk-js/src/ui/styledSystem/common.ts

```diff
@@ -97,17 +98,23 @@ const borderVariants = (t: InternalTheme, props?: any) => {
             borderColor: hoverBorderColor,
             WebkitTapHighlightColor: 'transparent',
             boxShadow: [
-              defaultBoxShadow,
-              t.shadows.$focusRing.replace('{{color}}', props?.hasError ? t.colors.$danger200 : t.colors.$primary50),
+              hoverBoxShadow,
+              t.shadows.$focusRing.replace(
+                '{{color}}',
+                !props?.hasError ? t.colors.$blackAlpha150 : t.colors.$danger200,
+              ),
             ].toString(),
           },
         };
   return {
     normal: {
       borderRadius: t.radii.$md,
       border: t.borders.$normal,
-      borderColor: !props?.hasError ? t.colors.$blackAlpha300 : t.colors.$danger300,
-      boxShadow: defaultBoxShadow,
+      borderColor: !props?.hasError ? t.colors.$blackAlpha150 : t.colors.$danger400,
```

Comment:
Should we use `dangerAlpha` here instead?

Reply:
I'll give it a try and see how it looks.

Reply:
Done. Also updated the PR's description to explain some changes I've made in the last commit.